app/domain/people_search.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+class PeopleSearch
+  SEARCHABLE_FIELDS = %w{name title competence_notes description
+                         role technology location}.freeze
+  attr_reader :search_term, :entries
+
+  def initialize(search_term)
+    @search_term = search_term
+    @entries = search_result
+  end
+
+  private
+
+  def search_result
+    people = Person.all.search(search_term)
+    people = pre_load(people)
+
+    people.map do |p|
+      { person: { id: p.id, name: p.name }, found_in: found_in(p) }
+    end
+  end
+
+  def found_in(person)
+    res = in_attributes(person.attributes)
+    res ||= in_associations(person)
+    res.try(:camelize, :lower)
+  end
+
+  # Load the attributes of the given people into cache
+  # Without this, reflective methods accessing attributes over associations
+  # would come up empty
+  def pre_load(people)
+    person_keys = people.map { |person| person.id }
```

Comment:
Sorry I forgot about this in my last review.
If you only call one method on each iteration object you can just use `people.map(&:id)`
That should do the trick